docs/guides/redirections-with-frontity.md

```diff
@@ -0,0 +1,156 @@
+# Redirections with Frontity
+
+Many users store their 30x redirections in the WordPress database, e.g. via a [Redirection plugin](https://wordpress.org/plugins/redirection/). WordPress redirects a user to a new link, for example after a user renames a post and tries to access it using the "old" link. However, no such functionality exists out-of-the-box for the REST API. Since Frontity uses the REST API to retrieve content, it needs a way of handling such redirections. 
+
+Have a look at this [Demo](#demo) for a bit more in-depth explanation.
```

Comment:
I think the link for the demo is not correct here.

Reply:
oh, nevermind - I think it's because of how github interprets the link. So, on gitbook the link should just point to the `#demo` section below, correct?